lite.auto.tfvars.example

```diff
@@ -0,0 +1,254 @@
+#-----------------------------------------------------------------------------
+
+# Name of the A record, of following dns zone, where the ecl watch ip is placed
+# This A record will be created and therefore should not exist in the following
+# dns zone.
+# Example entry: "my-product". This should be something project specific rather
+# than something generic.
+# Value type: string
+# Updateable: Y
+
+a_record_name="<eclwatch A record name>"
+
+#-----------------------------------------------------------------------------
+
+# Name of an existing dns zone.
+# Example entry: "hpcczone.us-hpccsystems-dev.azure.lnrsg.io"
+# Value type: string
+# Updateable: N
+
+aks_dns_zone_name="<dns zone name>"
+
+#-----------------------------------------------------------------------------
+
+# Name of the resource group of the above dns zone.
+# Example entry: "app-dns-prod-eastus2"
+# Value type: string
+# Updateable: N
+
+
+aks_dns_zone_resource_group_name="<dns zone resource group name>"
+
+#------------------------------------------------------------------------------
+
+# The version of HPCC Systems to install.
+# Only versions in nn.nn.nn format the 'latest' are supported.
+# Value type: string
+# Updateable: Y
+
+hpcc_version="9.4.4"
+
+#------------------------------------------------------------------------------
+
+# Enable ROXIE?
+# This will also expose port 8002 on the cluster.
+# Example entry: false
+# Value type: boolean
+# Updateable: Y
+
+aks_enable_roxie=false
+
+#------------------------------------------------------------------------------
+
+# Enable code security?
+# If true, only signed ECL code will be allowed to create embedded language functions, use PIPE(), etc.
+# Example entry: false
+# Value type: boolean
+# Updateable: Y
+
+enable_code_security=false
+
+#------------------------------------------------------------------------------
+
+# If you want a thor cluster then 'enable_thor' must be set to true
+# Otherwise it is set to false
+# Value type: boolean
+# Updateable: Y
+
+enable_thor=true
+
+#------------------------------------------------------------------------------
+
+# The number of Thor workers to allocate.
+# Must be 1 or more.
+# Value type: number
+# Updateable: Y
+
+thor_num_workers=2
+
+#------------------------------------------------------------------------------
+
+# The maximum number of simultaneous Thor jobs allowed.
+# Must be 1 or more.
+# Value type: number
+# Updateable: Y
+
+thor_max_jobs=2
```

Comment:
Let's change the default to 1 to avoid unnecessary resource allocation for people just kicking the tires.